### ISSUE TITLE
bundle utils disrupts installation of terminal42/contao-password-validation

### ISSUE THREAD

**Issue by zonky2** [heimrichhannot/contao-utils-bundle#19]
bundle utils disrupts installation of terminal42/contao password validation

with require terminal42/contao-password-validation send

`SQLSTATE[42S22]: Column not found: 1054 Unknown column 'j1.pwChangePage' in 'field list'`

```
        trace: {
          ./vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:78 { …}
          ./vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php:911 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Database/Statement.php:276 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Database/Statement.php:250 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Model.php:1088 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Model.php:959 { …}
          ./vendor/contao/core-bundle/src/Framework/Adapter.php:47 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Model/ModelUtil.php:103 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Cache/DatabaseTreeCache.php:90 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Cache/DatabaseTreeCache.php:78 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/DcaLoader.php:104 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:1261 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/dca/tl_article.php:11 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/DcaLoader.php:94 { …}
          ./vendor/contao/core-bundle/src/Resources/contao/library/Contao/Controller.php:1261 { …}
          ./vendor/contao/core-bundle/src/Framework/Adapter.php:47 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Dca/DcaUtil.php:943 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Cache/DatabaseTreeCache.php:251 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Cache/UtilCacheWarmer.php:132 { …}
          ./vendor/heimrichhannot/contao-utils-bundle/src/Cache/UtilCacheWarmer.php:84 { …}
          ./vendor/symfony/symfony/src/Symfony/Component/HttpKernel/CacheWarmer/CacheWarmerAggregate.php:52 { …}
          ./vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Command/CacheWarmupCommand.php:96 { …}
          ./vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:255 { …}
          ./vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:1005 { …}
          ./vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:86 { …}
          ./vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:255 { …}
          ./vendor/symfony/symfony/src/Symfony/Bundle/FrameworkBundle/Console/Application.php:74 { …}
          ./vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:148 { …}
          ./vendor/contao/manager-bundle/bin/contao-console:47 { …}
        }
```

after 

```
ALTER TABLE `tl_page` ADD `pwChangePage` INT(10) NOT NULL DEFAULT '0' ;
ALTER TABLE `tl_page` ADD `nc_account_disabled` INT(10) NOT NULL DEFAULT '0' ;
```

comes

`SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'type' in where clause is ambiguous`

**Comment by Defcon0** [comment]
Could you please provide the full sql leading to

SQLSTATE[23000]: Integrity constraint violation: 1052 Column 'type' in where clause is ambiguous

?

Which contao version is it?

**Comment by zonky2** [comment]
heimrichhannot/contao-request-bundle             1.1.0              A request class, that brings the symfony/http-foundat...
heimrichhannot/contao-teaser-bundle              0.3.2              A contao teaser content element, that provides the ab...
heimrichhannot/contao-utils-bundle               2.177.1            This bundle offers various utility functionality for ...
heimrichhannot/contao-video-bundle               0.1.4              This bundle brings enhanced video embedding support t...

Contao 4.9

**Comment by Defcon0** [comment]
Mhm, the question is where the WHERE constraint comes from. Do you have a stack trace after adding the fields to the db?

**Comment by zonky2** [comment]
this is the trace after ALTER TABLE

**Comment by Defcon0** [comment]
Should be fixed in 2.177.2. Could you please test if it works for you, as well now?

**Comment by zonky2** [comment]
works fine - thnks for your fast response!
